### ISSUE TITLE
"uv venv" doesn't find python (Linux)

### ISSUE THREAD

**Issue by BWindey** [astral-sh/uv#1335]
When trying out uv, I wanted to try install a package to the global space. Which isn't possible apparently, but ok, let's make a virtual environment like the command suggested to me: `uv venv`
Result:
```
    × Neither `python` nor `python3` are in `PATH`. Is Python installed?
```
Yes, Python is installed. Running `python3 --version` works. `which python3` returns `/usr/bin/python3`. `echo $PATH` returns `...:/usr/bin/:...` so python definitly is in my path. I don't really know what's going wrong here.

I'm on Zorin OS 17 (derivative of Ubuntu), current python version is 3.10.12.

Is there some more information I can give?

**Comment by zanieb** [comment]
Thanks for the report! I'll be looking into this. We've never seen this bug so I'll have to do some digging...

**Comment by zanieb** [comment]
Can you share the output of `uv venv -v`?

**Comment by zanieb** [comment]
Fixed in #1351

**Comment by BWindey** [comment]
Downloaded the new version and it works, awesome!
